plans/academic_module_3.markdown

```diff
@@ -0,0 +1,129 @@
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+1. Not really. Before the anchor system, we were all doing it and no one was responsible.
+2. Yes, Jeff. I took over when this became part of module 3.
+3. No. Steve and I started working on this for the last two modules.
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+1. We already have. Adding a third project and adding performance in the rubric.
+2. We already have. Adding two checkins per week and switching to agile.
+3. We already included this changes in the curriculum for the this last module.
+
+* What will be the next three things you do?
+
+1. Wait to see how the Feed Engine projects go and measure them.
+2. I think this is already successful.
+3. Continue to improve the JS curriculum and settle on a structure.
+
+* How will you keep track of progress and results?
+
+1. We included a metric to do so in the rubric.
+2. By the quality of the last projects.
+3. By how well their JS is. Maybe including some more in the final assessment of module 3.
+
+* Are you empowered to take the actions you think necessary?
+
+1. Yes, but rotation will make it difficult.
+2. Yes.
+3. Yes.
+
+## The Future
+
+* What would total success look like?
+
+1. Students are able to optimize their web applications, and use APIs with confidence.
+2. Students will be able to take a business need and transform it into product in a timely manner.
+3. Students will be able to utilize JavaScript and incorporate it in their projects.
+
+* What metrics can be used to evaluate results?
+
+1. Percentage of students that are able to obtain a 3 or more in performance in their final assessment.
+2. Percentage of students that are able to demo their individual projects in module 3.
+3. Percentage of students that obtain a 3 or more in their final assessment in JS.
+
+* What would meaningful progress look like in 3 months? 6 months? 12 months?
+
+1. 90 per cent or more students are able to obtain a 3 or more in performance in their final assessment.
+2. 90 per cent or more students are able to demo their individual projects in module 3.
+3. 90 per cent or more students obtain a 3 or more in their final assessment in JS.
+
+* How feasible are each of those goals?
+
+All of them are feasible.
+
+## The Periphery
+
+* What challenges are you going to stop working on to put time into this?
+
+I will keep working on what I am working since this is part of my responsibilities.
+
+* What research/learning do you need to do to be effective at solving it?
+
+1. I need to talk to more developers that have production experience.
+2. I will like to research and talk to more agile teams to get best practices. Maybe get certified in scrum.
+3. I will need to learn more JavaScript and a front-end framework such as Ember, React or Angular.
```

Comment:
I think for Module 3, that we may want to shy away from a framework. I like the idea of Backbone—even if it's not as popular these days. I think that—like Sinatra and Sequel—it's approachable and has some interesting ideas that students can apply to the way they write JavaScript.